deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/wrapper/OpenApiClientGeneratorWrapperTest.java

```diff
@@ -114,15 +99,72 @@ void circuitBreaker() throws URISyntaxException, FileNotFoundException {
     }
 
     private List<File> generateRestClientFiles() throws URISyntaxException {
-        Path targetPath = Paths.get(getTargetDir(), "openapi-gen");
-
-        Path simpleOpenApiFile = Path.of(Objects.requireNonNull(getClass().getResource("/openapi/simple-openapi.json"))
-                .toURI());
-
-        OpenApiClientGeneratorWrapper generatorWrapper = new OpenApiClientGeneratorWrapper(simpleOpenApiFile, targetPath)
+        OpenApiClientGeneratorWrapper generatorWrapper = createGeneratorWrapper("simple-openapi.json")
                 .withCircuitBreakerConfiguration(Map.of(
                         "org.openapitools.client.api.DefaultApi", List.of("opThatDoesNotExist", "byeGet")));
 
         return generatorWrapper.generate();
     }
+
+    private OpenApiClientGeneratorWrapper createGeneratorWrapper(String specFileName) throws URISyntaxException {
```

Comment:
I'm currently working on an issue that I did the same refactoring. Many thanks! Let's keep yours, I'll rebase after merging this PR.